### PR TITLE
Move to emotion

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

File: packages/website/babel.config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   presets: ['next/babel', '@babel/typescript'],
   plugins: [
+    'emotion',
     [
       'styled-components',
```
Comment (dpisani-atl) [inline packages/website/babel.config.js]: Should this be removed?

File: packages/website/components/navigation/all-packages-nav-content.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ import {
 } from '@atlaskit/navigation-next';
 import { gridSize as gridSizeFn } from '@atlaskit/theme';
 
+/** @jsx jsx */
```
Comment (dpisani-atl) [inline packages/website/components/navigation/all-packages-nav-content.js]: What does this comment do?

File: packages/website/babel.config.js (hunk 1 above)
Comment (Noviny) [reply packages/website/babel.config.js]: Not yet. We will still have a bunch of SC code on our site for some time - until AK updates and we update AK.

File: packages/website/components/navigation/all-packages-nav-content.js (hunk 2 above)
Comment (Noviny) [reply packages/website/components/navigation/all-packages-nav-content.js]: https://emotion.sh/docs/css-prop#jsx-pragma

File: packages/website/babel.config.js (hunk 1 above)
Comment (dpisani-atl) [reply packages/website/babel.config.js]: But surely the AK components are already transpiled. Isn't this SC babel plugin only for our own stuff?

File: packages/website/components/navigation/all-packages-nav-content.js (hunk 2 above)
Comment (dpisani-atl) [reply packages/website/components/navigation/all-packages-nav-content.js]: weird but ok

File: packages/website/babel.config.js (hunk 1 above)
Comment (Noviny) [reply packages/website/babel.config.js]: I'll try removing it and let you know.

Comment (Noviny) [reply packages/website/babel.config.js]: I removed it and everything broke, so leaving it in for now.

Comment (dpisani-atl) [reply packages/website/babel.config.js]: this makes me angery

**Pull request merged**